tests/e2e/map/map_spatial.spec.js

```diff
@@ -31,764 +52,1767 @@ test.describe.skip('Map: Spatial interactions', () => {
+
+          // Draws the spatial on the map
+          await expect(page).toHaveScreenshot('4326-circle-spatial-drawn.png', {
+            clip: screenshotClip
+          })
+        })
+      })
 
-        // Select the point spatial type
-        await page.getByRole('button', { name: 'spatial-selection-dropdown' }).click()
-        await page.getByRole('button', { name: 'Point' }).click()
+      test.describe('When typing a new circle in the spatial display', () => {
+        test('renders correctly', async ({ page }) => {
+          await interceptUnauthenticatedCollections({
+            page,
+            body: commonBody,
+            headers: commonHeaders,
+            additionalRequests: [{
+              body: circleBody,
+              headers: {
+                ...commonHeaders,
+                'cmr-hits': '2'
+              },
+              paramCheck: (parsedQuery) => parsedQuery?.circle?.[0]?.match(/42\.\d+,4\.\d+,156\d+/)
+            }]
+          })
+
+          await page.goto('/')
+
+          // Wait for the map to load
+          await page.waitForSelector('.edsc-map-base-layer')
+
+          // Select the circle spatial type
+          await page.getByRole('button', { name: 'spatial-selection-dropdown' }).click()
+          await page.getByRole('button', {
+            name: 'Circle',
+            exact: true
+          }).click()
+
+          // Enter the circle values
+          await page.getByTestId('spatial-display_circle-center').focus()
+          await page.keyboard.type('4.5297,42.1875')
+          await page.getByTestId('spatial-display_circle-radius').focus()
+          await page.keyboard.type('156444')
+          await page.keyboard.up('Enter')
+
+          // Updates the URL
+          await expect(page).toHaveURL(/search\?circle\[0\]=42.1875%2C4.5297%2C156444&lat=4\.\d+&long=42\.\d+&zoom=7\.\d+/)
+
+          // Populates the spatial display field
+          await expect(page.getByTestId('spatial-display_circle-center')).toHaveValue('4.5297,42.1875')
+          await expect(page.getByTestId('spatial-display_circle-radius')).toHaveValue('156444')
+
+          // Checking that the right number of results are loaded ensures that the route
+          // was fulfilled correctly with the succesfull paramCheck
+          await expect(page.getByText('Showing 2 of 2 matching collections')).toBeVisible()
+
+          // Wait for map animation to complete
+          await page.waitForTimeout(250)
+
+          // Draws the spatial on the map
+          await expect(page).toHaveScreenshot('4326-circle-spatial-typed.png', {
+            clip: screenshotClip
+          })
+        })
+      })
 
-        // Enter the spatial point
-        await page.getByTestId('spatial-display_point').focus()
-        await page.keyboard.type('4.5297,42.1875')
-        await page.keyboard.up('Enter')
+      test.describe.skip('When editing a circle', () => {
```

Comment:
```suggestion
      test.describe('When editing a circle', () => {
```

Reply:
We decided not to edit spatial right now, leaving these as skipped